cpp/src/Ice/SSL/SchannelEngine.cpp

```diff
@@ -1120,15 +1403,18 @@ namespace
 
 bool
 Schannel::SSLEngine::validationCallback(
+    HCERTCHAINENGINE chainEngine,
```

Comment:
I reworked the validation callback to be usable as the default validation callback. We can now use the same validation callback with IceSSL properties, and with auth options when the user doesn't provide one.